docs/appendices.txt

```diff
@@ -5,7 +5,7 @@
 [appendix]
 == API Documentation (JavaDoc)
 
-Refer to the JanusGraph API documentation for a complete documentation of all core APIs exposed by JanusGraph.
+Refer to the JanusGraph https://javadoc.io/doc/org.janusgraph/janusgraph-core[API documentation] for a complete documentation of all core APIs exposed by JanusGraph.
```

Comment:
For the record I think it would be better to link to the correct version as you suggest even if there's the error message for the snapshot case.